source/contracts/deployment.md

```diff
@@ -1,5 +1,5 @@
 # Deployment
-The Gnosis Safe smart contract was written with the usage of a proxy contract in mind. Because of that there is no constructor and it is required to call an inilize function on the contract before it can be used. For this it is recommended to use the ProxyFactory or the DelegatingConstructorProxy.
+The Gnosis Safe smart contract was written with the usage of a proxy contract in mind. Because of that there is no constructor and it is required to call an initilize function on the contract before it can be used. For this it is recommended to use the ProxyFactory or the DelegatingConstructorProxy.
```

Comment:
This should be `initialize`, right?

Thanks for pointing this out :)

Reply:
yes. yes it should :-) thanks!